solutionbox/image_classification/mltoolbox/image/classification/_predictor.py

```diff
@@ -95,7 +95,9 @@ class LoadImagesDoFn(beam.DoFn):
   """A DoFn that reads image from url."""
 
   def process(self, element):
-    with _util.open_local_or_gcs(element['image_url'], 'r') as ff:
+    from tensorflow.python.lib.io import file_io as tf_file_io
```

Comment:
There is no global "from tensorflow.python.lib.io import file_io as tf_file_io". Do local import in case Dataflow does not pickle the main session, just to be safe.